spec/features/chooser/chooser_disable_selected_options.rb

```diff
@@ -32,15 +32,26 @@
 
```

Comment:
Just noticed this file doesn't end with spec.rb,  I think we have a convention wherein all spec tests end with spec.rb